nat-lab/tests/test_lana.py

```diff
@@ -2019,6 +2021,22 @@ async def test_lana_same_meshnet_id_is_reported_after_a_restart(
             initial_beta_meshnet_id = beta_events[0].fp
 
             api.remove(beta.id)
+        if os.environ.get("NATLAB_SAVE_LOGS") is not None:
+            process = connection_beta.create_process(["cat", "./tcli.log"])
```

Comment:
I would recommend using `telio.get_logs()` as simply cat'ing logs while process is still running requires flushing.

Reply:
But we may merge it in separate PR, if we would like to merge this before nightly starts